docs/create-handlers.md

```diff
@@ -3,14 +3,326 @@ id: create-handlers
+****Check public scripts for examples.***
+2) **Rename the game exe:**
+Change the game exe name for each instance (in Nucleus-Coop = Game.ChangeExe = true;)
+3) **Change the title of the game window:**
+You can use https://www.nirsoft.net/utils/winexp.html for testing. (in Nucleus-Coop = Game.IdInWindowTitle = true;) 
+4) **Start the game as a different windows user:**
+For testing, you need a second windows user and press ***shift + right-click*** on the exe and choose ***"Rus as another user"***. (In Nucleus-Coop you can use Game.LaunchAsDifferentUsers = true; and if needed Game.ThirdPartyLaunch = true;)
 
-For more advanced tutorials on handlers and scripting, please check [Scripting](/docs/kbm-setup)
\ No newline at end of file
+If you use a script for testing, here two lines that could help you:
+
+- This line will keep the game files inside the Nucleus-Coop\content folder. In this way, you can do the process manually and verify if the problems you are getting are Nucleus-Coop related.  
+```Game.KeepSymLinkOnExit = true; 			//Enable or disable symlink files from being deleted when Nucleus is closed | default: false```
+
+- If by doing manually the second instance starts, while through Nucleus-Coop don't, try adding this line. (You can also disable all the not necessary lines and enable one at the time between each test to identify which one causes problems). 
+```Game.HookInit = true;					//Enable or disable hooks of functions some games may try and use to prevent multiple instances from running | default: false```
+
+### Connection between instances
+
+After you made multiple instances start, is the time to test the connectivity.
+While for games with native LAN you will just need to check if the instances see each other through the multiplayer menu, for games that must use Goldberg/Nemirtinga's emulators you will need to set them for each copy of the game (each instance must have a different SteamID inside user_steam_id.txt and nickname inside account_name.txt).
+I usually directly create a script and make Nucleus-Coop set the instances for me. Recently, a software for automatically set Goldberg for you was just released if you still want to try manually https://www.reddit.com/r/GoldbergEmu/comments/kxsaki/goldberggui_frontend_for_goldberg_emulator/?utm_source=share&utm_medium=web2x&context=3
+
+
+
+### Scripting Tips
+
+If all work is time to script it. Now, You will start testing input restriction and focus methods!
+Start by checking how the public scripts work and use as an example one that uses the same game-engine of the game you are trying to script.
+##### Some tips and essential lines
+You can find most of the lines in the Mod-ReadMe.txt (inside Nucleus-Coop root folder) and in the Master_SCript.js (documentation link inside Reddit FAQ post).
+Also, all my public scripts (Bizzo) have descriptions inside that could help you.
+
+Here some tips and essential lines:
+
+#### Game Info
+***These lines must be inside the script:***
```

Comment:
Change to H4, and like I mentioned earlier, the name should stay the same for now `#### Essential lines`